# x postion, in millimeters
float64 x
# y position, in millimeters
float64 y
# z position, in millimeters
float64 z
# z Euler component of object rotation, in radians
float64 angle_z
